Modules/_asynciomodule.c

```diff
@@ -646,6 +650,16 @@ future_get_result(FutureObj *fut, PyObject **result)
 
     fut->fut_log_tb = 0;
     if (fut->fut_exception != NULL) {
+        if (fut->fut_exception_tb != NULL) {
```

Comment:
Hi @erlend-aasland Would you like to review this? Thanks (there are some ref leaks)

Reply:
Hi, Kumar! I'll have a look hopefully in a couple of days.